examples/tools/scene_viewer.rs

```diff
@@ -187,10 +179,50 @@ fn keyboard_animation_control(
     }
 }
 
+fn keyboard_cameras_control(
+    mut cameras: Local<(bool, Vec<Entity>)>,
+    mut active_camera: Local<Option<usize>>, // `None` means user-controlled camera
+
+    scene_handle: Res<SceneHandle>,
+    keyboard_input: Res<Input<KeyCode>>,
+    camera_query: Query<Entity, (With<Camera3d>, Without<CameraController>)>,
+    user_camera_query: Query<Entity, (With<Camera3d>, With<CameraController>)>,
+    mut active_camera_3d: ResMut<ActiveCamera<Camera3d>>,
+) {
+    if !scene_handle.is_loaded {
+        return;
+    }
+    if !cameras.0 {
```

Comment:
I would prefer to use a simple struct here: the .0 and .1 is always very tricky to follow.